From 725ce6f9f9cb74398ed919e9e1614960dc2275f1 Mon Sep 17 00:00:00 2001
From: danie007 <danilselvan@gmail.com>
Date: Thu, 10 Jun 2021 13:14:09 +0530
Subject: [PATCH 2/3] Enable HAB features

1. Enable secure boot support
  - HAB status logs retrievement through the hab_status command
  - Support to extend the root of trust

2. Enable encrypted boot support

Signed-off-by: danie007 <danilselvan@gmail.com>
---
 configs/iot-gate-imx8_defconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/configs/iot-gate-imx8_defconfig b/configs/iot-gate-imx8_defconfig
index ca6a165732..eeb0b5512c 100644
--- a/configs/iot-gate-imx8_defconfig
+++ b/configs/iot-gate-imx8_defconfig
@@ -103,3 +103,13 @@ CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_EXT4_WRITE=y
 CONFIG_OF_LIBFDT_OVERLAY=y
 # CONFIG_EFI_LOADER is not set
+
+# Added by Yocto to support the HAB features
+# (Secure boot)
+CONFIG_IMX_HAB=y
+
+# Added by Yocto to support the HAB features
+# (Encrypted boot)
+CONFIG_CMD_DEKBLOB=y
+CONFIG_IMX_OPTEE_DEK_ENCAP=y
+CONFIG_CMD_PRIBLOB=y
